### ISSUE TITLE
Harden server side data downloads

### ISSUE THREAD

**Issue by andrewtackett** [memphis-iis/mofacts-ies#74]
Require authorization to download experiment data

**Comment by andrewtackett** [comment]
Also follow best practices listed here: https://guide.meteor.com/security.html

**Comment by imrryr** [comment]
what does this mean exactly? don't we already need to be logged in and this authorizes us?

**Comment by andrewtackett** [comment]
@imrryr Nope.  Downloads on the data downloads page for instance require no authorization.

**Comment by andrewtackett** [comment]
You would have to guess the link to download them but this is what is referred to as "security through obscurity" which is widely frowned upon

**Comment by imrryr** [comment]
Gotta fix this.... not acceptable and I wouldn't have agreed to it if I knew. There are regulations we are violating here.
